src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -194,253 +328,349 @@ const CurationPage = (props: ICurationPageProps) => {
+              <Row>
+                {oncogenicityFilter.map((filter, index) => (
+                  <Col className="col-6" key={filter.label}>
+                    <InputGroup>
+                      <Input
+                        onChange={() => handleFilterCheckboxChange(index, setOncogenicityFilter)}
+                        checked={filter.selected}
+                        style={{ marginLeft: '0px' }}
+                        type="checkbox"
+                      />
+                      <Label style={{ marginLeft: '18.75px' }}>{filter.label}</Label>
+                    </InputGroup>
+                  </Col>
+                ))}
+              </Row>
+              <h6 className="mb-2 mt-2">Mutation effect</h6>
+              <Row>
+                {mutationEffectFilter.map((filter, index) => (
+                  <Col className="col-6" key={filter.label}>
+                    <InputGroup>
+                      <Input
+                        onChange={() => handleFilterCheckboxChange(index, setMutationEffectFilter)}
+                        checked={filter.selected}
+                        style={{ marginLeft: '0px' }}
+                        type="checkbox"
+                      />
+                      <Label style={{ marginLeft: '18.75px' }}>{filter.label}</Label>
+                    </InputGroup>
+                  </Col>
+                ))}
+              </Row>
+              <h6 className="mb-2 mt-2">Therapeutic implication level of evidence</h6>
+              <Row className="align-items-start justify-content-start">
+                {txLevelFilter.map((filter, index) => (
+                  <Col style={{ flexGrow: 0 }} key={filter.label}>
+                    <InputGroup>
+                      <Input
+                        onChange={() => handleFilterCheckboxChange(index, setTxLevelFilter)}
+                        checked={filter.selected}
+                        style={{ marginLeft: '0px' }}
+                        type="checkbox"
+                      />
+                      <Label style={{ marginLeft: '18.75px' }}>{filter.label}</Label>
+                    </InputGroup>
+                  </Col>
+                ))}
+              </Row>
+              <Row className="justify-content-end">
+                {isFiltered() && (
+                  <Col className="px-0 mr-2" style={{ flexGrow: 0 }}>
+                    <Button
+                      outline
+                      color="danger"
+                      onClick={() => {
+                        setOncogenicityFilter(initFilterCheckboxState(oncogenicityFilterOptions));
+                        setMutationEffectFilter(initFilterCheckboxState(mutationEffectFilterOptions));
+                        setTxLevelFilter(initFilterCheckboxState(txLevelFilterOptions));
+                        setShowFilterModal(false);
+                      }}
+                    >
+                      Reset
+                    </Button>
```

Comment:
Wonder if we should also have a reset button next to the text `Showing x of y matching search`.